src/Wizard/Wizard.component.js

```diff
@@ -5,14 +5,17 @@ import ToolbarGroup from 'material-ui/Toolbar/ToolbarGroup';
 import RaisedButton from 'material-ui/RaisedButton/RaisedButton';
 import Card from 'material-ui/Card/Card';
 import CardText from 'material-ui/Card/CardText';
+import IconButton from 'material-ui/IconButton';
+import HelpOutlineIcon from 'material-ui/svg-icons/action/help-outline';
+import Dialog from 'material-ui/Dialog/Dialog';
+import FlatButton from 'material-ui/FlatButton/FlatButton';
 import Steps from 'react-steps';
 
 const Wizard = React.createClass({
     mixins: [Translate],
 
     propTypes: {
         fields: React.PropTypes.arrayOf(React.PropTypes.object),
-        buttons: React.PropTypes.arrayOf(React.PropTypes.object),
```

Comment:
`this.props.buttons` is referenced in line 149, what's the idea of removing the propType?